recipe/meta.yaml

```diff
@@ -1,43 +1,56 @@
-{% set version = "302" %}
+{% set name = "pywin32" %}
+{% set version = "305" %}
+{% set build = "0" %}
+{% set checksum = "a6d7a5c5578ad0bbb4a8f9cf6ff277612dab5c21a614ea243a5bf72882f4cbca" %}
 
 package:
-  name: pywin32
+  name: {{ name }}
   version: {{ version }}
 
 source:
   url: https://github.com/mhammond/pywin32/archive/b{{ version }}.tar.gz
-  sha256: 47fd53c0561ff3019043e86880597aacbd736910d8add485117d8ad055441927
+  sha256: {{ checksum }}
   patches:
-    - disable_exchange_modules.patch
-    - 0003-remove-PATH-pth-hack.patch
-    # https://github.com/mhammond/pywin32/commit/9fbc2ed3c0d69aca2475975a599cc316016599d3
-    # - fix-formatting.patch
-    # HACK: Avoid https://bugs.python.org/issue42009
-    # - ensure-PythonService-mc-before-cpp.patch
-    # - add-verbosity-to-setup-py.patch
+    - patches/0001-remove-PATH-pth-hack.patch
+    - patches/0002-MSVC_dirs_setup.patch
+    - patches/0003-MSVC_dirs_setup_fix.patch
 
 build:
-  number: 2
+  number: {{ build }}
   skip: True  # [not win]
+  missing_dso_whitelist:
+    - $RPATH/axscript.pyd
+    - $RPATH/win32ui.pyd
+    - $RPATH/win32uiole.pyd
 
 requirements:
   build:
     - {{ compiler("c") }}
-    - m2-patch  # [win]
-    - m2-gcc-libs  # [win]
-    - patch  # [not win]
+    - m2-patch
+    - m2-gcc-libs
   host:
     - python
     - pip
     - setuptools
   run:
     - python
 
+test:
+  files:
+    - test_win32gui.py
+  requires:
+    - pip
+    - python
+  commands:
+    - pip check
+    - python run_test.py
```

Comment:
An error on `win64`:
```
  File "C:\b\abs_627wkvj_i7\croot\pywin32_1669249312158\test_tmp\run_test.py", line 53, in <module>

    assert os.path.isfile(pythoncom_filename)

AssertionError
```

Reply:
If you add to run_test.py:
```
import glob

print(
    list(
        glob.glob(
            os.path.join(
                os.environ["LIBRARY_BIN"],
                'py*.dll'
            )
        )
    )
)
```
the result will be the empty list `[]`

Reply:
The latest added patch solves this issue.